modules/ROOT/pages/connected-apps-devs-end-users.adoc

```diff
@@ -19,4 +17,4 @@ They can can authorize access to their account when the following message appear
 
 // screenshot of authorize user page https://qax.anypoint.mulesoft.com/login/authorize?state=eyJjbGllbnQiOnsiY2xpZW50X2lkIjoiZWEzYzJiODFkMDEyNGI0Mjk1NDYyM2RhMWM0MzQ5ZWIifSwidXNlciI6eyJ1c2VybmFtZSI6Im9yZ2FuaXphdGlvbl9vd25lciJ9LCJvcmlnaW4iOiIvYXBpL3YyL29hdXRoMi9hdXRob3JpemU%2FY2xpZW50X2lkPWVhM2MyYjgxZDAxMjRiNDI5NTQ2MjNkYTFjNDM0OWViJnJlZGlyZWN0X3VyaT1odHRwczovL2V4YW1wbGUuY29tJnJlc3BvbnNlX3R5cGU9dG9rZW4mc2NvcGU9ZnVsbCIsInRyYW5zYWN0aW9uX2lkIjoiSXlNVGVJSHciLCJzY29wZSI6WyJmdWxsIl19
 
-End users can manage third-party apps that they have authorized in their profile settings. To view autorized applications, click on the user's icon on the top right and navigate to profile. 
+End users can manage third-party apps that they have authorized in their profile settings. The user can view authorized applications in their profile.
```

Comment:
Change "applications" to "apps." 
"The user" does not match "their" in the last sentence. Why did you remove the navigation information for viewing the apps?

Reply:
```suggestion
End users can manage third-party apps that they have authorized in their profile settings. The user can view authorized apps in their profile.
```

Reply:
Users know how to access their profiles, presumably, so I would prefer not to restate it.